pyart/retrieve/cappi.py

```diff
@@ -0,0 +1,195 @@
+"""
+Constant Altitude Plan Position Indicator
+"""
+
+import numpy as np
+from netCDF4 import num2date
+from pandas import to_datetime
+from scipy.interpolate import RectBivariateSpline
+
+from pyart.core import Radar
+
+
+def create_cappi(
+    radar,
+    fields=None,
+    height=2000,
+    gatefilter=None,
+    same_nyquist=True,
+    nyquist_vector_idx=0,
+):
+    """
+    Create a Constant Altitude Plan Position Indicator (CAPPI) from radar data.
+
+    Parameters:
+    -----------
+    radar : Radar
+        Py-ART Radar object containing the radar data.
+    fields : list of str, optional
+        List of radar fields to be used for creating the CAPPI.
+        If None, all available fields will be used (default is None).
+    height : float, optional
+        The altitude at which to create the CAPPI (default is 2000 meters).
+    gatefilter : GateFilter, optional
+        A GateFilter object to apply masking/filtering to the radar data (default is None).
+    same_nyquist : bool, optional
+        Whether to only stack sweeps with the same Nyquist velocity (default is True).
+    nyquist_vector_idx : int, optional
+        Index for the Nyquist velocity vector if `same_nyquist` is True (default is 0).
+
+    Returns:
+    --------
+    Radar
+        A Py-ART Radar object containing the CAPPI at the specified height.
+
+    Notes:
+    --------
+    CAPPI (Constant Altitude Plan Position Indicator) is a radar visualization
+    technique that provides a horizontal view of meteorological data at a fixed altitude.
```

Comment:
@mgrover1 Thoughts on this? I'm on the fence with citing wikipedia

Reply:
Agreed!

Reply:
I suggest using the AMS glossary here https://glossary.ametsoc.org/wiki/Cappi